lib/gold_miner/gold_nugget.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module GoldMiner
+  GoldNugget = Data.define(:content, :author, :source) do
```

Comment:
Let me know if I'm leaning into the gold miner analogy too much. I also considered calling this `GoldOre`.